### DISCUSSION TITLE
使用浏览器引入的方式，希望开启明细表排序

### DISCUSSION THREAD

**Discussion by xujiamu** [antvis/S2#2731] [Q&A] [answered]
![image](https://github.com/antvis/S2/assets/53904639/df837619-eda7-45ee-85e0-0415ad2599d4)

![image](https://github.com/antvis/S2/assets/53904639/e288649d-a2e2-4152-934e-76b5943cb70a)

如上配置，只能出现图标，点击没反应,请问如何调整
![image](https://github.com/antvis/S2/assets/53904639/f55cff9c-9d9a-4318-9d4a-7ef9c163f989)

**Comment by lijinke666** [comment, marked as answer]
排序菜单是 `@antv/s2-react` 封装的, 你使用的是 `@antv/s2`, 所以点了没反应, 需要 [自定义 Tooltip, 自行实现 UI](https://s2-v1.antv.antgroup.com/manual/basic/tooltip#%E6%B3%A8%E6%84%8F%E4%BA%8B%E9%A1%B9), 然后调用 `@antv/s2` 提供的[排序 API](https://s2.antv.antgroup.com/manual/basic/sort/group)

**Reply by lijinke666** [reply]
增加一个 Demo, 你可以参考下 https://github.com/antvis/S2/pull/2733/files#diff-73edda65f80482b349c1100b2cbe3f68211294f0660619171a898554c374700fR51-R91

**Reply by xujiamu** [reply]
尝试了一下，1.x版本 CDN 没有s2.groupSortByMethod 方法， CDN 改为2.0后，Demo可用， 感谢
